model/src/rollover.rs

```diff
@@ -5,6 +5,12 @@ use crate::Price;
 use crate::TxFeeRate;
 use crate::Usd;
 
+#[derive(Debug, Clone, Copy)]
+pub enum Version {
+    V1_0_4_7,
+    Current,
```

Comment:
Just make these 1 and 2?

I think this should version the rollover protocol itself, not which version of the daemon this is implemented.

Also "current" is not a version :)

Reply:
This enum would also be a good place to document the difference in the 2 versions of the protocol!

Reply:
I agree with what you are pointing out and will rename to `V1` and `V2` but probably still mention in rustdoc for what version `V1` was for if that's OK. 

I feel "if this grows", i.e. if we at some point have multiple version it would be great to know for what version we introduced them. That could make it easier to monitor what to remove when we know that there are no takers running 0.4.7 anymore.

Reply:
this sounds like a good idea - in fact, this would be in line with what I've already done for `HelloV2` message (rustdoc comment that `Hello` was used by takers up to 0.4.7).